modules/detectron/CMakeLists.txt

```diff
@@ -19,6 +20,17 @@ if (BUILD_CAFFE2_OPS)
     if (MSVC)
       install(FILES $<TARGET_PDB_FILE:caffe2_detectron_ops_gpu> DESTINATION lib OPTIONAL)
     endif()
+  elseif(USE_ROCM)
+    set(Caffe2_HIP_INCLUDES ${Caffe2_HIP_INCLUDES} ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Add current dir to include dirs list is not desired, instead we should change the includes in the source files to use path relative to the project root (i.e. `#include "modules/detectron/xxx.h"` instead of `#include "xxx.h"`)